xml/ay_bigfile.xml

```diff
@@ -4490,6 +4490,42 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
    </para>
 
    <sect2 xml:id="Software.Selections.sles10">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You've to
+     choose which product you wish to install. You can do it explicitly using.
+     <literal>product</literal> option
+     <example>
+       <title>Explicit product selection</title>
+       <para>
+         &lt;software&gt;
+           &lt;products&gt;
+             &lt;product&gt;
+               SLED15
```

Comment:
Please use a single line, to avoid whitespace problems: `&lt;product&gt;SLED15&lt;/product&gt;`
(note the slash, missing in your example)

Reply:
fixed, thx